docs/toast.rst

```diff
@@ -104,10 +104,14 @@ When running workflows in batch mode, we typically use a slightly different
 structure from a notebook.  There are helper functions in toast to parse
 operator options from the commandline and / or a config file, and to automatically
 create named instances of those operators for use in workflow.  There is
-a large-scale workflow in ``sotodlib/workflows/toast_so_sim.py``.  This
+a large-scale workflow in ``sotodlib/workflows/toast_so_map.py``.  This
```

Comment:
I think you mixed `toast_so_map.py` and `toast_so_sim.py` on line 111.

Reply:
Yes, clarified and fixed.